vendor/blockchain_contracts/src/bin/print_offsets.rs

```diff
@@ -0,0 +1,25 @@
+use blockchain_contracts::{
+    ethereum::rfc003::{Erc20Htlc, EtherHtlc},
+    format_table,
+};
+
+#[allow(clippy::print_stdout)]
+fn main() {
+    println!("### RFC003 ###");
+
+    {
+        println!("** Ether on Ethereum **");
+        let contract = EtherHtlc::compile_template_to_hex();
+        println!("Contract template:\n {}", contract);
+        let offsets = EtherHtlc::get_all_offsets();
+        println!("{}", format_table(offsets));
```

Comment:
:wrench: It is not a super critical part of the software but from a responsibility point of view, the formatting should IMO be done in here and not in the library.

If you want to go for _formatted_ text output, I would prefer MD so that we can just copy it into the RFC (to avoid mistakes).
Alternatively, you could just `Debug` print it.

Reply:
Happy to action this, please answer the other comment regarding your enum idea.

Reply:
❌ Yes do this. IMO the goal for this thing is to export nothing as a library and just be a cli to generate hex.

Reply:
Please check latest changes and review.